src/main/java/at/hannibal2/skyhanni/data/PetData.kt

```diff
@@ -0,0 +1,15 @@
+package at.hannibal2.skyhanni.data
+
+import at.hannibal2.skyhanni.utils.LorenzRarity
+import at.hannibal2.skyhanni.utils.NEUInternalName
+
+data class PetData(
+    val internalName: NEUInternalName,
+    val cleanName: String,
+    val rarity: LorenzRarity,
+    val petItem: NEUInternalName,
+    val hasSkin: Boolean,
```

Comment:
can we instead save the pet skin and still check if it has a skin by doing a null check

Reply:
the issue with this is you can't get the skin from the autopet or the tab widget, you can only get it's rarity

Reply:
I understand it so that we either have the skin data (non-null string) or we have no skin data (skin is null). In that case, we don't need an extra variable if the skin is available.